apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -60,6 +60,26 @@ function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 	return true;
 }
 
+/**
+ * Get the WPCOM blog id of the current site for tracking purposes.
+ */
+function wpcom_global_styles_get_wpcom_current_blog_id() {
```

Comment:
I initially thought about it too, however, this function has a different behavior for disconnected Atomic sites: instead of passing the id = 1, it returns null (since that's the expected value for tracking purposes). In the end, I opted for simplicity and duplicated the code.

I guess we could have a condition where it checks if the id returned is `1`. If it's 1, then we can assume that the blogId returned should be passed as `null`. 

I can make the change if you want, but I think it would give double meaning to the blogId variable (in the current state): sometimes is the WPCOM blog id(most of the time), but sometimes is `1`, the local id of the site (for the disconnected sites).